front/src/service.ts

```diff
@@ -76,21 +77,41 @@ export interface User {
     rooms: string[]
 }
 
+interface ServerToClientEvents {
+    message: (msg: Message) => void,
+    // TODO: rename that
+    people: (user: User) => void,
```

Comment:
change `User` to `User[]` its my bad.

Reply:
fixes here [dc3d561](https://github.com/Draeggon/scratchy/pull/2/commits/dc3d561cfc5e1729b32b00c7ad94cd8b448c7c5d)